http_2.rflx

```diff
@@ -0,0 +1,238 @@
+                  or ((Frame_Type = DATA
+                     or Frame_Type = SETTINGS
+                     or Frame_Type = PING
+                     or Frame_Type = GOAWAY
+                     or Frame_Type = RST_STREAM
+                     or Frame_Type = WINDOW_UPDATE
+                     or Frame_Type = PRIORITY
+                  ) and Flag_End_Headers = False));
+         Flag_Padded : Boolean
+            then Bit_5_Unused
+               if (Frame_Type = DATA
+                  or Frame_Type = HEADERS
+                  or Frame_Type = PUSH_PROMISE
+                  or ((Frame_Type = SETTINGS
+                     or Frame_Type = PING
+                     or Frame_Type = CONTINUATION
+                     or Frame_Type = GOAWAY
+                     or Frame_Type = RST_STREAM
+                     or Frame_Type = WINDOW_UPDATE
+                     or Frame_Type = PRIORITY
+                  ) and Flag_Padded = False));
+         Bit_5_Unused : Zero_Bit;
+         Flag_Priority : Boolean
+            then Bit_7_Unused
+               if (Frame_Type = HEADERS
+                  or ((Frame_Type = DATA
+                     or Frame_Type = SETTINGS
+                     or Frame_Type = PING
+                     or Frame_Type = GOAWAY
+                     or Frame_Type = RST_STREAM
+                     or Frame_Type = WINDOW_UPDATE
+                     or Frame_Type = PRIORITY
+                     or Frame_Type = PUSH_PROMISE
+                     or Frame_Type = CONTINUATION
+                  ) and Flag_Priority = False));
+         Bit_7_Unused : Zero_Bit;
+         Bit_8_Unused : Zero_Bit;
+         Reserved_Bit_1 : Zero_Bit;
+         Stream_Identifier : Stream_ID    -- Last Header Field
+            then Pad_Length
+               if (Frame_Type = DATA or Frame_Type = HEADERS or Frame_Type = PUSH_PROMISE) and Flag_Padded = True
+            then Data
+               with Size => Payload_Length * 8
+               if Frame_Type = DATA and Flag_Padded = False
+            then Exclusive_Flag
+               if Frame_Type = HEADERS and Flag_Priority = True and Flag_Padded = False
+            then Header_Block_Fragment
+               with Size => Payload_Length * 8
+               if (Frame_Type = HEADERS and Flag_Priority = False and Flag_Padded = False) or Frame_Type = CONTINUATION
+            then Exclusive_Flag
+               if Frame_Type = PRIORITY
+            then Error_Code
+               if Frame_Type = RST_STREAM
+            then Settings_Array
+               with Size => Payload_Length
+               if Frame_Type = SETTINGS
+            then Data
+               with Size => 64
+               if Frame_Type = PING and Stream_Identifier = 0
+            then Reserved_Bit_2
+               if (Frame_Type = GOAWAY and Stream_Identifier = 0)
+                  or Frame_Type = WINDOW_UPDATE
+                  or (Frame_Type = PUSH_PROMISE and Flag_Padded = False);
+         Settings_Array : Settings_Array
```

Comment:
Couldn't this field and type just be named `Settings`?

Reply:
Renamed in 7d43ab6